### ISSUE TITLE
Pihole -up fail on system with no web interface installed

### ISSUE THREAD

**Issue by fmjensen** [pi-hole/pi-hole#1871]
**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*.
- [?] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the the source code relevant to this issue?:**

`{7}`

---
**Expected behaviour:**

` update success `

**Actual behaviour:**

`update fails after web interface check`

**Steps to reproduce:**

`update system that have opted out of web interface installation`

**Debug token provided by [uploading `pihole -d` log](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debug):**

`n/a`

**Troubleshooting undertaken, and/or other relevant information:**

`execute pihole -up twice`
![0417d991-438a-4e49-820b-e16556e01bfd](https://user-images.githubusercontent.com/3484684/34325555-33059084-e895-11e7-975b-2df786e947b2.jpeg)
![9a526423-62cc-4d96-bddf-a2042d911e15](https://user-images.githubusercontent.com/3484684/34325556-332096fe-e895-11e7-8c09-3a84aa5e9faf.jpeg)

**Comment by fmjensen** [comment]
Pinhole... Duh... Bloody autocorrect 🤬

**Comment by PromoFaux** [comment]
Would you mind sharing a debug token? Or at least the contents of `/etc/pihole/setupVars.conf`? If the latter, feel free to redact any information. I'm mainly interested to see what the value `INSTALL_WEB` is set to in that file...

**Comment by PromoFaux** [comment]
OK, set my value to `false` and removed the web directory.. looks like we have a bug!

```
fap@fappotron:~$ sudo nano /etc/pihole/setupVars.conf
fap@fappotron:~$ sudo rm -rf /var/www/html/admin
fap@fappotron:~$ pihole -up
  [i] Checking for updates...
  [i] Pi-hole Core:     up to date
  [i] FTL:              up to date

  [i] Everything is up to date!
fap@fappotron:~$ pihole -r

        .;;,.
        .ccccc:,.
         :cccclll:.      ..,,
          :ccccclll.   ;ooodc
           'ccll:;ll .oooodc
             .;cll.;;looo:.
                 .. ','.
                .',,,,,,'.
              .',,,,,,,,,,.
            .',,,,,,,,,,,,....
          ....''',,,,,,,'.......
        .........  ....  .........
        ..........      ..........
        ..........      ..........
        .........  ....  .........
          ........,,,,,,,'......
            ....',,,,,,,,,,,,.
               .',,,,,,,,,'.
                .',,,,,,'.
                  ..'''.


  [✓] Root user check
  [i] Repair option selected
  [✓] Disk space check

  [✓] Update local cache of available packages

  [✓] Checking apt-get for upgraded packages... 9 updates available
  [i] It is recommended to update your OS after installing the Pi-hole!

  [i] Installer Dependency checks...
  [✓] Checking for apt-utils
  [✓] Checking for dialog
  [✓] Checking for debconf
  [✓] Checking for dhcpcd5
  [✓] Checking for git
  [✓] Checking for iproute2
  [✓] Checking for whiptail

  [i] Performing reconfiguration, skipping download of local repos
  [✓] Resetting repository within /etc/.pihole...
  Unable to reset /var/www/html/admin, exiting installer
```

**Comment by fmjensen** [comment]
You ate too fast for me @PromoFaux 😂
Here you have a token any ways :
zrtt6rw9z2
Merry Christmas 🎄

**Comment by dschaper** [comment]
Fixed in #1943
